Extension/package.json

```diff
@@ -154,6 +154,18 @@
           "minimum": 0.1,
           "maximum": 1
         },
+        "C_Cpp.inactiveRegionFontColor": {
+          "type": "string",
```

Comment:
The type should accept "null" as valid, since that's the default, e.g.
[
            "string",
            "null"
],